src/test/java/org/openlmis/requisition/domain/requisition/RequisitionLineItemDataBuilder.java

```diff
@@ -68,6 +68,11 @@ public class RequisitionLineItemDataBuilder implements DataBuilder<RequisitionLi
   //this needs to be always idealStockAmount - stockOnHand
   private Integer calculatedOrderQuantityIsa = 50;
   private Integer additionalQuantityRequired = 0;
+  private Integer numberOfPatientsOnTreatmentNextMonth;
```

Comment:
Why this doesn't have a default, but the others do?

Reply:
I added the value.